platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -183,6 +205,53 @@ public void run() {
     });
   }
 
+  /**
+   * Merge offline regions from a secondary database into the main offline database.
+   * <p>
+   * When the merge is completed, or fails, the {@link MergeOfflineRegionsCallback} will be invoked on the main thread.
+   * <p>
+   * The secondary database may need to be upgraded to the latest schema.
+   * This is done in-place and requires write-access to the provided path.
+   * <p>
+   * Only resources and tiles that belong to a region will be copied over. Identical
+   * regions will be flattened into a single new region in the main database.
+   * <p>
+   * A callback with a `MapboxOfflineTileCountExceededException` error will be invoked if
```

Comment:
How/Where is this handled? I don't see any reference to this exception type anywhere else.

Reply:
Good catch, we pushing the error through `MergeOfflineRegionsCallback#onError`. The doc needs to be updated.